distrobox-enter

```diff
@@ -217,8 +217,9 @@ generate_command() {
 	# Entering container using our user and workdir.
 	# Start container from working directory. Else default to home. Else do /.
 	# pass distrobox-enter path, it will be used in the distrobox-export tool.
+	workdir=$(printf "%s" "${PWD:-${HOME:-/}}" | sed 's/./\\&/g')
 	result_command="${result_command}
-		--workdir=${PWD:-${HOME:-"/"}}
+		--workdir=${workdir}
```

Comment:
Seems to be enough to just quote (escaping) the workdir and it works with spaces and special chars:

`--workdir=\"${PWD:-${HOME:-"/"}}\"`

Reply:
I'll correct myself, this won't cover 1 use-case, the presence of quotes in the path name. using this should solve our prolems:

```sh
	workdir="$(echo "${PWD:-${HOME:-"/"}}" | sed -e 's/"/\\\"/g' )"
	result_command="${result_command}
		--workdir=\"${workdir}\"
```

Reply:
@89luca89 Thank you for your reply.

It works perfectly.